### DISCUSSION TITLE
Arbor documentation review

### DISCUSSION THREAD

**Discussion by brenthuisman** [OCNS/SoftwareWG#14] [General]
We've made some effort in shaping up the [Arbor documentation](https://arbor.readthedocs.io/en/latest/). If you have some time and would like to review the Arbor documentation, please leave your input [here](https://docs.google.com/forms/d/e/1FAIpQLSfxaEOmws0BLeWGCcavgHyQOo_gbuuWsWDynk6MxsxlnH54Lw/viewform). Especially outside perspectives welcome!

**Comment by sanjayankur31** [comment]
I'm not sure if the team gets notifications unless they're mentioned in the discussion. So, let's try this: cc: @OCNS/software-wg
